teku/src/main/java/tech/pegasys/teku/cli/util/SlashingProtectionCommandUtils.java

```diff
@@ -24,7 +26,7 @@ public static void verifySlashingProtectionPathExists(
       final SubCommandLogger subCommandLogger, final Path slashProtectionPath) {
     if (!slashProtectionPath.toFile().exists() || !slashProtectionPath.toFile().isDirectory()) {
       subCommandLogger.exit(
-          1,
+          FATAL_EXIT_CODE,
```

Comment:
actually i think this is the last command tool, so may as well say this one too...